google-cloud-spanner-hibernate-dialect/src/main/java/com/google/cloud/spanner/hibernate/SpannerDialect.java

```diff
@@ -329,8 +329,8 @@ public String getCurrentSchemaCommand() {
 
   @Override
   public SchemaNameResolver getSchemaNameResolver() {
-    throw new UnsupportedOperationException(
-        "No schema name resolver supported by " + getClass().getName());
+    // Spanner does not have a notion of database name schemas, so return "".
```

Comment:
Throw the behavior change into release notes?

Reply:
This change should be a no-op; I upgraded the version of hibernate in the PR and it seems like newer version requires us to return a schema resolver instead of throwing `UnsupportedOperationException()`. In either case though this object does not affect the user.